config/initializers/extensions/active_record.rb

```diff
@@ -1 +0,0 @@
-require 'active_record_extension'
```

Comment:
Can this entire file be deleted? Or maybe we should auto-load everything in `lib/extensions`?

Reply:
Yes because its being loaded in ApplicationRecord. Now all models inherit.